src/docs/asciidoc/user-guide/assertj-core-release-notes.adoc

```diff
@@ -280,13 +280,13 @@ Examples:
 [source,java]
 ----
 // assertions succeed
-assertThat(42L).doesNotHaveToString(2501L);
-assertThat("The Force").doesNotHaveToString(null);
+assertThat(42L).doesNotHaveSameHashCodeAs(2501L);
+assertThat("The Force").doesNotHaveSameHashCodeAs("Awakens");
```

Comment:
NullPointerException is thrown when the other is null.

```sh
java.lang.NullPointerException: The object used to compare actual's hash code with should not be null
  at java.base/java.util.Objects.requireNonNull(Objects.java:233)
  at org.assertj.core.internal.Objects.assertDoesNotHaveSameHashCodeAs(Objects.java:931)
  at org.assertj.core.api.AbstractAssert.doesNotHaveSameHashCodeAs(AbstractAssert.java:998)
```

Reply:
Good catch! The javadoc also mentions that `NullPointerException` is thrown if `other` is null.